src/app/modules/reports/components/time-entries-table/time-entries-table.component.html

```diff
@@ -1,10 +1,6 @@
 <div class="row scroll-table mt-5 ml-0">
-  <table
-    class="table table-striped mb-0"
-    datatable
```

Comment:
Could you please restore this multiline format, it is more readable than the two lines.